app/jobs/scan_image_job.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+require 'tty-command'
+
+class ScanImageJob < ApplicationJob
+  def perform(imagestream, profile, b64_identity)
+    @image_name = imagestream.name
+    @registry_url = imagestream.openshift_connection.registry_api_url
+    @namespace, @image_name = image_name.split('/')
+    @profile = profile
+
+    download_image(imagestream)
+    run_oscap_docker(imagestream)
+
+    ParseReportJob.perform_later(
+      File.read(report_filepath),
+      imagestream.openshift_connection.account.account_number,
+      b64_identity
+    )
+  end
+
+  def download_image(imagestream)
+    Docker.authenticate!(
+      'username' => imagestream.openshift_connection.username,
+      'password' => imagestream.openshift_connection.token,
+      'serveraddress' => @registry_url
+    )
+    Docker::Image.create(
```

Comment:
So... is this creating a new image in the customer's openshift? Or is this local to our pod?

Reply:
This is local. Perhaps the library naming is weird, but if you track `journalctl -lf -u docker` while you run this, you will see it just downloads the image.